### ISSUE TITLE
[Bug]: Wrong link texts by using App "Two-Factor Authentication via Nextcloud notification"

### ISSUE THREAD

**Issue by schlagi123** [nextcloud/desktop#4504]
### ⚠️ Before submitting, please verify the following: ⚠️

- [X] This is a **bug**, not a question or a configuration issue.
- [X] This issue is **not** already reported on Github (I've searched it).
- [X] Nextcloud Server and Desktop Client are **up to date**. See [Server Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) and [Desktop Releases](https://nextcloud.com/install/#install-clients) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/)

### Bug description

I use the App "Two-Factor Authentication via Nextcloud notification" in the nextcloud. When I get the notification from the App I get 2 links with the same Text (Marked as readed). The left link accepts the authentication and the right cancel the authentication (Onley the displayed texts of the links are wrong).
![wrong-notification](https://user-images.githubusercontent.com/5476837/167311405-c2a43008-ae7d-422b-9558-1c1d2b19a63a.PNG)

### Steps to reproduce

Prerequisites:
- App "Two-Factor Authentication via Nextcloud notification" is installed on server
- User has setup Two-Factor Authentication via Nextcloud notification.

Steps to reproduce:
1. Login in on loginscreen in the Browser via Username and Password.
2. Choose Two-Factor Authentication via Nextcloud notification on next screen
3. Look for Notification in the Desktop App

### Expected behavior

I see one link text for accept two factor authentication and another to deny or cancel authentication.

### Which files are affected by this bug

There is no affected file, this is a display issue

### Operating system

Windows

### Which version of the operating system you are running.

Windows 10

### Package

Other

### Nextcloud Server version

23.0.3.2

### Is this bug present after an update or on a fresh install?

Updated to a major version (ex. 3.3.6 to 3.4.0)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### Are you using an external user-backend?

- [X] Default internal user-backend
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

**Comment by langfingaz** [comment]
I have the same problem. My nextcloud-client version is:

```shell
$ nextcloud --version
Nextcloud version 3.5.0git
Git revision ccb79dc69f97ef08b08d57fb53933bf2da4a22e4
Using Qt 5.15.3, built against Qt 5.15.3
Using Qt platform plugin 'xcb'
Using 'OpenSSL 1.1.1o  3 May 2022'
Running on Arch Linux, x86_64

```

**Comment by camilasan** [comment]
This was fixed with #4518, it is available in 3.5.1.
